flow/project.py

```diff
@@ -1940,6 +2426,44 @@ def _generate_operations(self, cmd, jobs, requires=None):
             cmd_ = cmd.format(job=job)
             yield JobOperation(name=cmd_.replace(' ', '-'), cmd=cmd_, job=job)
 
+    def _gather_flow_groups(self, names=None):
+        """Grabs FlowGroups that match names"""
+        operations = OrderedDict()
+        # if no names are selected try all singleton groups
+        names = names if names is not None else self._operations.keys()
+        for name in names:
+            if name in operations.keys():
+                continue
+            groups = [group for gname, group in self.groups.items() if
+                      re.fullmatch(name, gname)]
+            if groups != list():
+                for group in groups:
+                    operations[group.name] = group
+            else:
+                # remove error for now until desided
+                # raise ValueError("Operation/Group {} is not defined".
+                #                  format(name))
+                pass
```

Comment:
The original idea was if a name was specified that had no matches.

Reply:
The current behavior if you pass a name `'op_does_not_exist'` no error is raised. I originally thought it should error since the user might expect it to match something. However, this matches currently behavior so I will just remove those lines.